sequencer/sequencesender.go

```diff
@@ -215,7 +215,7 @@ func (s *Sequencer) handleEstimateGasSendSequenceErr(
 }
 
 func isDataForEthTxTooBig(err error) bool {
-	return strings.Contains(err.Error(), errGasRequiredExceedsAllowance) ||
+	return errors.Is(err, ethman.ErrGasRequiredExceedsAllowance) ||
 		errors.Is(err, core.ErrOversizedData) ||
-		strings.Contains(err.Error(), errContentLengthTooLarge)
+		errors.Is(err, ethman.ErrContentLengthTooLarge)
```

Comment:
```suggestion
		errors.Is(err, etherman.ErrContentLengthTooLarge)
```

Reply:
this is not valid as it collides with the interface inside the package: https://github.com/0xPolygonHermez/zkevm-node/blob/develop/sequencer/interfaces.go#L32